src/main/java/org/springframework/data/web/SortHandlerMethodArgumentResolverSupport.java

```diff
@@ -293,6 +295,19 @@ static boolean notOnlyDots(String source) {
 		return StringUtils.hasText(source.replace(".", ""));
 	}
 
+	@Nullable
+	private static String getQualifier(@Nullable MethodParameter parameter) {
```

Comment:
@mp911de, since this piece of code duplicates the equally named method from `PageableHandlerMethodArgumentResolverSupport`, we could consider moving it to some utility class.

Not sure whether `SpringDataAnnotationUtils` is the right place, due to its (possibly outdated) javadoc?

Reply:
Yes, it makes sense. I'll take care of this during the merge.